### DISCUSSION TITLE
How can I get Mew to obey?

### DISCUSSION THREAD

**Discussion by MJB19286200** [haven1433/HexManiacAdvance#242] [Q&A]
Does anyone know how I can get Mew to obey if I catch Mew in wild for Pokemon Fire Red? I know it can be done because in Pokemon Ultra Violet Mew obeys.

**Comment by themayonaiseman12-create** [comment]
setvar var4 151
  setvar var5 5
  setvar var6 0
  special CreateEventLegalEnemyMon
  setflag 0x08C1
  special BattleSetup_StartLegendaryBattle
